p2p_wallet/Scenes/Main/BankTransfer/WithdrawCalculator/WithdrawCalculatorCoordinator.swift

```diff
@@ -56,14 +58,24 @@ final class WithdrawCalculatorCoordinator: Coordinator<WithdrawCalculatorCoordin
                     break
                 }
             })
-            .map({ result -> PendingTransaction? in
+            .asyncMap({ result -> (WithdrawCoordinator.Result, TokenPrice?) in
+                let priceService = Resolver.resolve(PriceService.self)
+                let prices = try? await priceService.getPrice(
```

Comment:
how long it might be before price will appear? is it a api request?

Reply:
It should be a really fast, since it's always in cache